test/git-open.bats

```diff
@@ -4,6 +4,7 @@ load "test_helper/bats-support/load"
 load "test_helper/bats-assert/load"
 
 foldername="sandboxrepo"
+sshconfig=~/.ssh/config
```

Comment:
perhaps instead of this we could allow a env variable to override the sshconfig path.  that way your real config isn't touched during testing. (i've definitely run into problems when creating the test suite and exiting it halfway ;)

wdyt?

Reply:
👍  Although ~/.ssh is one of the few apps that you generally can't override, see https://wiki.archlinux.org/index.php/XDG_Base_Directory_support and https://bugzilla.mindrot.org/show_bug.cgi?id=2050

Reply:
Yeah, I don't think it's possible to get around this. If we want it to be super safe, I can have it only copy in the sshconfig if none already exists, and otherwise just skip all related tests. Would that be preferred?

Reply:
Actually, I guess there isn't any reason that the sshconfig be in the "right" place when testing, same as overriding BROWSER with `echo`, we can just supply it to the executable on runtime. I'll change that.